lib/src/Base/Graph/Polygon.cxx

```diff
@@ -169,4 +169,29 @@ void Polygon::load(Advocate & adv)
   adv.loadAttribute( "edgeColor_", edgeColor_ );
 }
 
+/* Builds a polygon which fills the area between two curves */
+Polygon Polygon::FillBetween(Sample const& dataX, Sample const& dataY1, Sample const& dataY2)
+{
+  return FillBetween(dataX.asPoint(), dataY1.asPoint(), dataY2.asPoint());
+}
+
+/* Builds a polygon which fills the area between two curves */
+Polygon Polygon::FillBetween(Point const& dataX, Point const& dataY1, Point const& dataY2)
+{
+  const UnsignedInteger size = dataX.getSize();
+  if ((dataY1.getSize() != size) || (dataY2.getSize() != size))
+    throw InvalidArgumentException(HERE) << "Error: cannot fill between curves based on numerical samples with different size.";
```

Comment:
numerical samples -> samples

Reply:
Apparently, this dates back all the way to the original "Import 1.6" GitHub commit, so yes, I imagine this message references the old NumericalSample class.

By the way, was there ever a non-numerical sample class?

Reply:
Fixed now, anyway.